tests/test_pyglmnet.py

```diff
@@ -133,6 +135,17 @@ def test_group_lasso():
     scaler = StandardScaler().fit(Xr)
     glm_group.fit(scaler.transform(Xr), yr)
 
+    # count number of nonzero coefs for each group.
+    # in each group, coef must be [all nonzero] or [all zero].
+    group_ids = np.unique(groups)
+    for group_id in group_ids:
+        if group_id == 0:
+            continue
+
+        target_beta = beta[groups == group_id]
+        n_nonzero = (target_beta != 0.0).sum()
+        assert n_nonzero in (len(target_beta), 0)
```

Comment:
I like the test as it's compact. But the problem is that it passes on `master`. Ideally, you want the test to _fail_ on `master` branch so the bug does not happen again